packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -150,122 +211,108 @@ class _MyHomePageState extends State<MyHomePage>
-                // AdaptiveScaffold for these types of navigations but this
-                // navigation has custom staggered item animations.
-                child: NavigationRail(
-                  onDestinationSelected: (int index) {
-                    setState(() {
-                      _navigationIndex = index;
-                    });
-                  },
-                  selectedIndex: _navigationIndex,
-                  leading: ScaleTransition(
-                    scale: _controller1,
-                    child: const _ComposeIcon(),
-                  ),
-                  backgroundColor: const Color.fromARGB(0, 255, 255, 255),
-                  labelType: NavigationRailLabelType.none,
-                  destinations: <NavigationRailDestination>[
-                    slideInNavigationItem(
+              builder: (_) =>
+                  // Usually it would be easier to use a builder from
+                  // AdaptiveScaffold for these types of navigations but this
+                  // navigation has custom staggered item animations.
+                  AdaptiveScaffold.standardNavigationRail(
+                onDestinationSelected: (int index) {
+                  setState(() {
+                    _navigationIndex = index;
+                  });
+                },
+                selectedIndex: _navigationIndex,
+                leading: ScaleTransition(
+                    scale: _controller1, child: const _MediumComposeIcon()),
+                backgroundColor: const Color.fromARGB(0, 255, 255, 255),
+                labelType: NavigationRailLabelType.none,
+                destinations: <NavigationRailDestination>[
+                  slideInNavigationItem(
                       begin: -1,
                       controller: _controller,
                       icon: Icons.inbox,
-                      label: 'Inbox',
-                    ),
-                    slideInNavigationItem(
+                      label: 'Inbox'),
+                  slideInNavigationItem(
                       begin: -2,
                       controller: _controller1,
                       icon: Icons.article_outlined,
-                      label: 'Articles',
-                    ),
-                    slideInNavigationItem(
+                      label: 'Articles'),
+                  slideInNavigationItem(
                       begin: -3,
                       controller: _controller2,
                       icon: Icons.chat_bubble_outline,
-                      label: 'Chat',
-                    ),
-                    slideInNavigationItem(
+                      label: 'Chat'),
+                  slideInNavigationItem(
                       begin: -4,
                       controller: _controller3,
                       icon: Icons.video_call_outlined,
-                      label: 'Video',
-                    )
-                  ],
-                ),
+                      label: 'Video')
```

Comment:
Nit: Sorry, just caught one last non-obvious one. Commas after the label strings to fix indentations.